mod_bot/gym.py

```diff
@@ -0,0 +1,111 @@
+import Levenshtein
+
+candidates_key = "candidates"
+
+
+def get_approx_name(gym_name, gym_list):
+    candidates_list = []
+    try:
+        gn = clean_name(gym_name)
+
+        # Try autocomplete
+        keys = gym_list.keys()
+        gyms = list(map(lambda x: clean_name(x), keys))
+
+        candidates = [gym for gym in gyms if gn in gym]
+        candidates = list(map(lambda x: find_gym_from_clean_name(x, gyms, gym_list), candidates))
+        if len(candidates) == 1:
+            return candidates[0]
+
+        candidates_list.extend(candidates)
+
+        # Try short_name
+        short_gyms = list(map(lambda x: clean_name(x[0]), gym_list.values()))
+        candidates = [gym for gym in short_gyms if gn in gym]
+        candidates = list(map(lambda short_name: find_gym_from_short_name(short_name, gym_list), candidates))
```

Comment:
The call to `find_gym_from_short_name` seems useless or too complicated, we already have the cleaned list of short names, no need to redo computations on gym_list to find an exact match

Reply:
this is under the asumption that a shortname at the beginning of the method is cleaned, but this is dumb I will modify it

Reply:
welp never mind, it's not useless but I can make it easier